### ISSUE TITLE
best and simple method to link IF_prompt_MKR to Oobabooga

### ISSUE THREAD

**Issue by marc2608** [if-ai/IF_prompt_MKR#14]
Hi, I don't have much programming experience, but I'm starting to get the hang of A1111. I'm very interested in IF prompt MKR and I've installed it in my A1111. However, I can't get it to work with Oobabooga, which I've also installed and which works very well but only as a text generation webui, with no link yet to the IF prompt MKR extension. I would have liked to find a step-by-step explanation of how to make them work together to create prompts. For example, a simple explanation of the API and how to insert it in the IF prompt MKR settings in A1111, the flags, the basic parameters of oobabooga in order to make it work with the MKR extension etc... 
Thank you in advance!

**Comment by if-ai** [comment]
Yes I have done a few tutorials not sure if you have seen my channel it's been a long time I haven't use A1111 so it might have changed a lot since. I will try to make a new tutorial soon

**Comment by marc2608** [comment]
Thank you very much for your answer. I really want to use this beautiful extension, but I'm having a recurring problem. Can you help me by giving me your opinion? These are the messages I get, both in automatic1111 when I use IF prompt MKR and in the cmd window when I open A1111. It's the same with Oobabooga as with Ollama; please help me correct this problem if you have some time!
![Capture d'écran 2024-06-28 182214](https://github.com/if-ai/IF_prompt_MKR/assets/132713311/2c1b5a00-0372-4c7c-9f3c-7bee8c2b29dd)
![Capture d'écran 2024-06-28 182108](https://github.com/if-ai/IF_prompt_MKR/assets/132713311/75593174-1de9-42b9-9098-f81f964fd2cd)

**Comment by if-ai** [comment]
Is basic but do you have ollama installed already? You can go into ollama.com and install Ollama for your system then reboot make sure you download a model if you want to use SD you can he this model by opening your terminal and typing `ollama run impactframes/llama3_ifai_sd_prompt_mkr_q4km:latest` that will download a model that is s special for SD prompts then when it finishes start A1111 make sure the if prompt maker settings has IP of your machine normally is 127.0.0.1 and the port is 11434
If you have a VPN you might need to allow the IP communication your machine might also have another IP different than 127.0.0.1 in which case you need to find the number you can type ipconfig on a CMD Window

**Comment by marc2608** [comment]
Yes, Ollama and model are installed. The model appears in the list and here are the extension settings in A1111. 
My A1111 version is: 
version: v1.9.4 - python: 3.10.6 - torch: 2.1.2+cu121 - xformers: 0.0.23.post1 - gradio: 3.41.2
Maybe it is not compatible? 
![Capture d'écran 2024-06-28 191418](https://github.com/if-ai/IF_prompt_MKR/assets/132713311/3b89c142-1930-46d2-b5dd-ef26ea8416e0)
![Capture d'écran 2024-06-28 191131](https://github.com/if-ai/IF_prompt_MKR/assets/132713311/25d30828-b4e5-42d8-8853-7aa9a00279de)
